pages/blog/miniseries-part_1.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Mini series - Speed up your development process (part 1)"
+date: 2021-04-28T06:00:00+01:00
+type: Engineering
+tags:
+  - tools
+  - code generation
+cover: /img/posts/jonaslagoni-miniseries-part1/blog-miniseries-cover.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+featured: true
+---
+
+How can you utilize code generation to speed up the development process and only focus on what is important, the business logic? In this mini-series we will explore the ways AsyncAPI and code generation can work hand in hand beyond generating documentation. 
```

Comment:
I suggest you always stick to "I" and as little as possible to "we". This is not documentation but YOUR article, it is better when you write in the first person, as it sounds personal